inc/parser/xhtml.php

```diff
@@ -1108,7 +1108,13 @@ function emaillink($address, $name = null, $returnonly = false) {
     function internalmedia($src, $title = null, $align = null, $width = null,
                            $height = null, $cache = null, $linking = null, $return = false) {
         global $ID;
-        list($src, $hash) = explode('#', $src, 2);
+
+        if (strpos($src, '#') !== false) {
+            list($src, $hash) = explode('#', $src, 2);
+        } else {
+            $hash = null;
+        }
+
```

Comment:
This deteriorate the readability of the code... 
Or as workaround, it doesn't help to prefix it with a `@` to suppress the notice... (not a real solution of course)
(why could `explode($delimiter, $string [, $limit ] )` not return always 2 entries if you use `$limit=2` ? Interesting feature request for PHP imho)